### DISCUSSION TITLE
Restoring backup created with older longhorn release

### DISCUSSION THREAD

**Discussion by luifrancisco** [longhorn/longhorn#8079] [Q&A]
Hi! I don't think it's anywhere mentioned if there is any longhorn version dependency when restoring backups. I am currently running v1.5.1 and would like to create a new cluster with the latest 1.6.0 release and was planning to restore the backup from the older version. While I've already tested this at home and can say it works fine by being able to access the data from the application after restoration, I am not sure if there will be any risk down the line with this method. Any help is appreciated! Thanks.

**Comment by derekbit** [comment]
No worries. The backup format works with any version.
